amazon-lex-connector-demo/src_hd/com/axonivy/connector/amazon/lex/demo/ChatBot/ChatBotProcess.mod

```diff
@@ -84,45 +84,20 @@ Cs0 f6 bodyObjectType com.amazonaws.lex.runtime.v2.SessionIdTextBody #txt
 Cs0 f6 bodyObjectMapping 'param.text=in.input;
 ' #txt
 Cs0 f6 resultType com.amazonaws.lex.runtime.v2.RecognizeTextResponse #txt
-Cs0 f6 responseCode 'import com.amazonaws.lex.runtime.v2.RecognizeTextResponseIntent;
-import com.amazonaws.lex.runtime.v2.RecognizeTextResponseInterpretations;
-import com.axonivy.connector.amazon.lex.demo.Message;
-import java.util.Map;
-if (result.#messages is initialized)
-{
+Cs0 f6 responseCode 'import com.amazonaws.lex.runtime.v2.Intent;
+import com.amazon.lex.connector.BotResponse;
```

Comment:
Why use a different namespace here? This code is from us and not from amazon. I therefore suggest to use com.axonivy.connector.amazon.lex.connector package name.

Reply:
Yes you are right: I thought more in terms of features than of the author and tried to avoid long namespaces. 
However, it's now adjusted and matches better to existing namespaces in the connector.